tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -47,6 +70,27 @@ func NewShowSchedulerCommand() *cobra.Command {
 	return c
 }
 
+// NewResumeSchedulerCommand returns a command to resume a scheduler.
+func NewResumeSchedulerCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "resume <scheduler>",
+		Short: "resume a scheduler",
+		Run:   resumeSchedulerCommandFunc,
+	}
+	return c
+}
+
+func resumeSchedulerCommandFunc(cmd *cobra.Command, args []string) {
```

Comment:
`pauseSchedulerCommandFunc` and `resumeSchedulerCommandFunc` have the same logic. We can use a common function?